support/codelabs/canonicai/main.md

```diff
@@ -0,0 +1,1589 @@
+# Build a conversational AI system in Jaseci
```

Comment:
This main.md contains all the content from chapter 1- 7md. We can remove chapter 1- 7.